sdk/apimanagement/api-management-custom-widgets-tools/review/api-management-custom-widgets-tools.api.md

```diff
@@ -38,7 +40,13 @@ export function buildBlobDataPath(name: string): string;
 export function buildOnChange<Values extends ValuesCommon>(): OnChange<Values>;
 
 // @public
-export function deployNodeJS(serviceInformation: ServiceInformation, name: string, fallbackConfigPath?: string, rootLocal?: string): Promise<void>;
+export type DeployConfig = {
+    rootLocal?: string;
+    interactiveBrowserCredentialOptions?: InteractiveBrowserCredentialNodeOptions;
+};
+
+// @public
+export function deployNodeJS(serviceInformation: ServiceInformation, name: string, fallbackConfigPath?: string, { rootLocal, interactiveBrowserCredentialOptions, }?: DeployConfig): Promise<void>;
```

Comment:
if we're changing this function signature in a breaking way anyway, should `fallbackConfigPath` get moved into the options bag?

We tend to like to keep all optional arguments together.

Reply:
In the default content produced by the scaffolder, the `fallbackConfigPath` is prepopulated. So changing it would break Deploy for pretty much every user. Meanwhile `rootLocal` is not and it's mostly for debugging, so I guess 99 % users never used that.
This is why I decided move only one of them, let me know if you'd still suggest to move both.

Reply:
I trust your instincts, I think the main reason we group options is to avoid needing to write a literal `undefined` for an optional param if you don't want to pass it but do want a different option in the option bag. But you know your customers better than I do.

Reply:
Yes, unnecessary undefineds is exactly why I'm doing this change now. If this was a new code, I'd do how you suggest but to avoid annoying customers, I'd keep it as is. :)